test/profile/both.def.exp2

```diff
@@ -0,0 +1,5 @@
+
+FUNCTIONS
+	_Dmain
```

Comment:
You have tabs here ... (you need to either replace them or make the check [more specific](https://github.com/dlang/druntime/blob/master/posix.mak#L279))..